### DISCUSSION TITLE
How to Sign Up/Create an Account in Redash

### DISCUSSION THREAD

**Discussion by Tomski23** [getredash/redash#6998] [Q&A]
Can you guide me on how to sign up for a Redash account? The get started page of Redash is too complicated for a beginner; it doesn't provide a basic understanding of how to create an account.

**Comment by lucydodo** [comment]
Hi @Tomski23, Would you have read this too? https://redash.io/help/open-source/setup
If you're still struggling with this, let me know and I'll explain a bit more. :)

**Reply by Tomski23** [reply]
Yes, could you please explain step-by-step how to do it? I want to create
an account without using open source. Does Redash have easy settings for
making an account?  Thanks!

**Reply by lucydodo** [reply]
What do you mean by not using open source? Do you mean you want to use a pre-hosted platform?

**Reply by Tomski23** [reply]
I mean the easy way to create an account, similar to Metabase, where I was able to quickly set up an account.

**Reply by lucydodo** [reply]
In this case, @justinclift's comment below will help you.

**Comment by justinclift** [comment]
@Tomski23 Are you looking to run Redash yourself, or are you wanting to use a version that's [managed by someone else](https://newdash.io/)?

**Reply by Tomski23** [reply]
The company I work for would like to trial Redash to evaluate its
capabilities. We are beginners, so is there an easy access guide on how to
create an account in Redash that doesn't use open source?

**Reply by justinclift** [reply]
Understood.  Please add yourself at the sign-up page here: https://newdash.io

There's still a bunch of manual processes that triggers, and someone (probably me) will get in contact with you within the day and go through potential pricing, (etc).

That being said, what are the types of data sources you're wanting to pull data from, and are the people who'd be trying it out in your company familiar with writing "SQL queries"?

In most cases writing SQL queries is a skill that will be needed, though it does depend if your data sources offer a SQL interface. :smile:
